app/controllers/recipes_controller.rb

```diff
@@ -59,6 +59,7 @@ def update
 
   # DELETE /recipes/1 or /recipes/1.json
   def destroy
+    redirect_to :show, notice: 'This user is not allowed to delete this recipe.' unless authorize @recipe
```

Comment:
I guess I should have commented on this line as well. You can remove it because authorize is already being called and the redirect won't happen because it raises an error

Reply:
I just missed it. I looked around for any other authorize. My bad.